docs/reference/indices/index-templates.asciidoc

```diff
@@ -15,7 +15,10 @@ template may still match and be applied.
 ====
 
 [[getting]]
-An index template is a way to tell {es} how to configure an index when it is created. Templates are
+An index template is a way to tell {es} how to configure an index when it is created.
+For data streams, the index template configures the stream's backing indices as they
+are created.
+Templates are
```

Comment:
Slightly weird spacing here, but not a big deal :) maybe we can turn it into a haiku?